content/docs/apps/experimental/private-egress.md

```diff
@@ -0,0 +1,76 @@
+---
+menu:
+  docs:
+    parent: experimental
+title: Communicating with private external services
+---
+
+You can set up all, or just a subset, of your applications to be able communicate privately and securely with external services in your agency infrastructure or with other cloud providers.
+
+## How it works
+
+Normally, any application deployed to cloud.gov gets placed onto one, or in the case of multiple instances, many servers or cells. These cells are shared by all applications using containers for resource separation, allocation, and security. This also means they share the internal routes of egress and firewalling to offered services, like databases.
+
+cloud.gov has the ability to work with you to establish an isolated set of servers, or cells, where only your Organization can place applications that need access to services that you control on other networks. Your applications will still be accessible from the Internet and can still connect to any of our offered internal services, just like any other applications on the platform, but they will have additional access to services on your networks. These networks will be connected with a Virtual Private Network (VPN) using industry standard Internet Protocol Security (IPSec) protocol. We can work with you to establish the number of required isolated servers to handle your workload of applications that would need this type of access to your networks and services.
+
+{{< diagrams id-prefix="colocated-diagram" >}}
+graph LR;
```

Comment:
I would rather not have to scroll to see the whole graph. WDYT about switching to `graph TB`?

Reply:
Switched, though it's not much better. Still have to scroll. Also, added ASG information.